mvicore/src/main/java/com/badoo/mvicore/feature/BaseFeature.kt

```
@@ -120,12 +143,17 @@ open class BaseFeature<Wish : Any, in Action : Any, in Effect : Any, State : Any
         }
     }
 
-    private class ActorWrapper<State : Any, Action : Any, Effect: Any>(
-        private val threadVerifier: SameThreadVerifier,
-        private val disposables: DisposableCollection,
+    private operator fun CompositeDisposable.plusAssign(any: Any?) {
```

Should we raise an exception if this is not a disposable?

---

I am a little worried to add it here, never checked how it actually works.
The current implementation is copied from `DisposableCollection` which I replaced with build-in `CompositeDisposable`. `StandaloneMiddleware` is `Disposable`, but other middlewares might not be. I am not ready to address this issue in this PR.

---

Fair enough